src/sbr/easinTemplates.py

```diff
@@ -40,10 +40,10 @@
 spring:
   profiles: dev
   datasource:
-    url: jdbc:h2:/tmp/{{project.name}}/sbr-gen-database.h2;DB_CLOSE_ON_EXIT=FALSE
+    url: jdbc:postgresql://localhost:5432/essDB
     username: easin
```

Comment:
instead of Easin let's use the project name :  

```suggestion
    username: {{project.name}
```

Reply:
you did not add this ?